Using `google.protobuf.BytesValue` generates Uint8Array rather than Buffer for Node env

Sample proto file with the field type of `google.protobuf.BytesValue`

sample.proto
```
message Message {
  google.protobuf.BytesValue file = 1;
}
```

Generating the TS files for the above proto using the below command creates the type as `Uint8Array`
`protoc --plugin=./node_modules/.bin/protoc-gen-ts_proto --ts_proto_opt=env=node --ts_proto_out=./src/libs/`

Note: As per mentioned in the library doc I am passing the env as node in options (https://www.npmjs.com/package/ts-proto#supported-options)

sample.ts

```
export interface Message {
	file: Uint8Array | undefined;
}
```

Compiling the build is throwing the below error

```
src/libs/sample.ts:21:7 - error TS2322: Type 'Uint8Array' is not assignable to type 'Buffer'.

21     { value: message.file! },
         ~~~~~

  src/libs/google/protobuf/wrappers.ts:94:2
    94  value: Buffer;
        ~~~~~
    The expected type comes from property 'value' which is declared here on type 'BytesValue'
```

--- a/src/types.ts
+++ b/src/types.ts
@@ -422,7 +422,7 @@ export function valueTypeName(ctx: Context, typeName: string): Code | undefined 
     case '.google.protobuf.BoolValue':
       return code`boolean`;
     case '.google.protobuf.BytesValue':
-      return code`Uint8Array`;
+      return ctx.options.env === EnvOption.NODE ? code`Buffer` : code`Uint8Array`;
     case '.google.protobuf.ListValue':
       return code`Array<any>`;
     case '.google.protobuf.Value':

